app/assets/javascripts/legacy/views/option_set_form.js

```diff
@@ -166,9 +166,6 @@
 
     // show/hide the option levels field
     self.option_levels_field.show(checked)
-
-    // enable/disable nested options
-    self.options_field.list.allow_nesting(checked)
```

Comment:
This isn't something that should depend on whether levels are enabled or not.

Reply:
Won't this disable nested options? Note this is being called on `options_field` and not `option_levels_field`.

Reply:
ah you are right, i put it back in, and the other fix still works!